src/vcpkg/binarycaching.cpp

```diff
@@ -31,42 +31,44 @@ namespace
                                  (msg::count, msg::vendor),
                                  "",
                                  "Attempting to fetch {count} package(s) from {vendor}");
-    DECLARE_AND_REGISTER_MESSAGE(RestoredPackagesFromVendor,
-                                 (msg::count, msg::elapsed, msg::vendor),
-                                 "",
-                                 "Restored {count} package(s) from {vendor} in {elapsed}");
-    DECLARE_AND_REGISTER_MESSAGE(UploadedPackagesToVendor,
-                                 (msg::count, msg::elapsed, msg::vendor),
-                                 "",
-                                 "Uploaded {count} package(s) to {vendor} in {elapsed}");
-    DECLARE_AND_REGISTER_MESSAGE(UnknownBinaryProviderType,
-                                 (),
-                                 "",
-                                 "unknown binary provider type: valid providers are 'clear', 'default', 'nuget', "
-                                 "'nugetconfig', 'interactive', and 'files'");
-    DECLARE_AND_REGISTER_MESSAGE(InvalidArgumentRequiresPrefix,
-                                 (msg::binary_source),
+    DECLARE_AND_REGISTER_MESSAGE(
+        AuthenticationMayRequireManualAction,
+        (msg::vendor),
+        "",
+        "One or more {vendor} credential providers requested manual action. Add the binary source "
+        "'interactive' to allow interactivity.");
+    DECLARE_AND_REGISTER_MESSAGE(CompressFolderFailed, (msg::path), "", "Failed to compress folder '{path}':");
+    DECLARE_AND_REGISTER_MESSAGE(
+        DefaultPathToBinaries,
+        (msg::path),
+        "",
+        "Based on your system settings, the default path to store binaries is \n '{path}'. This consults "
+        "%LOCALAPPDATA%/%APPDATA% on Windows and $XDG_CACHE_HOME or $HOME on other platforms.");
+    DECLARE_AND_REGISTER_MESSAGE(ExtendedDocumenationAtUrl,
+                                 (msg::url),
                                  "",
-                                 "invalid argument: binary config '{binary_source}' requires at least a prefix");
-    DECLARE_AND_REGISTER_MESSAGE(InvalidArgumentRequiresSource,
-                                 (msg::binary_source),
+                                 "Extendend documentation available at '{url}'.");
```

Comment:
I think it's OK to just put the URL into the localized message, IMO. But I think we are inconsistent about that at the moment...

Reply:
I am hoping to be able to reuse some of these messages in the future. Passing the URL in as an argument allows me to do that.

Reply:
Hmmmmm I don't know if we have another place that tries to do that but sounds OK

Reply:
Actually, I just realized that InvalidArgumentRequiresSource is repetitive since there is already an InvalidArgumentRequiresSourceArgument message (which is also used several times throughout the file).